packages/venia-concept/src/components/Header/header.js

```diff
@@ -6,6 +6,9 @@ import classify from 'src/classify';
 import Icon from 'src/components/Icon';
 import CartTrigger from './cartTrigger';
 import NavTrigger from './navTrigger';
+import SearchTrigger from './searchTrigger';
+
+import SearchBar from 'src/components/SearchBar';
```

Comment:
This seems like a good candidate for lazy loading—with [`React.lazy()`](https://github.com/magento-research/pwa-studio/blob/release/2.0/packages/venia-concept/src/components/MiniCart/miniCart.js#L18) and [`Suspense`](https://github.com/magento-research/pwa-studio/blob/release/2.0/packages/venia-concept/src/components/MiniCart/miniCart.js#L128-L130).